t/opcode-coverage.neon

```diff
@@ -0,0 +1,594 @@
+FUNCTION ned(): Boolean
+    RETURN dictionary <> dictionary
+END FUNCTION
+
+#    EQP,        // compare equal pointer
+FUNCTION eqp()
+    IF NIL = NIL THEN
+    END IF
+END FUNCTION
+
+#    NEP,        // compare unequal pointer
+FUNCTION nep()
+    IF NIL <> NIL THEN
+    END IF
+END FUNCTION
+
+#    EQV,        // compare equal voidptr
+FUNCTION eqv()
+    VAR p1: Handle
+    p1 := NIL
+    IF p1 = NIL THEN
+    END IF
+END FUNCTION
+
+#    NEV,        // compare unequal voidptr
+FUNCTION nev()
+    VAR p1: Handle
+    p1 := NIL
+    IF p1 <> NIL THEN
+    END IF
+END FUNCTION
+
+#    ANDB,       // and boolean
+FUNCTION andb(): Boolean
+    RETURN TRUE AND FALSE
+END FUNCTION # Does not generate opcode in Neon binaries.
+
+#    ORB,        // or boolean
+FUNCTION orb(): Boolean
+    RETURN TRUE OR FALSE
+END FUNCTION # Does not generate opcode in Neon binaries.
+
+#    NOTB,       // not boolean
+FUNCTION notb(): Boolean
+    RETURN NOT TRUE
+END FUNCTION
+
+#    INDEXAR,    // index array for read
+FUNCTION indexar(): Number
+    RETURN array[0]
+END FUNCTION
+
+#    INDEXAW,    // index array for write
+FUNCTION indexaw()
+    array[42] := 42
+END FUNCTION
+
+#    INDEXAV,    // index array value
+FUNCTION indexav()
+    FUNCTION a(): Array<Number>
+        RETURN array
+    END FUNCTION
+    print("\(a()[1])")
+END FUNCTION
+
+#    INDEXAN,    // index array value, no exception
+FUNCTION indexan(): String
+    RETURN rec().field
+END FUNCTION
+
+#    INDEXDR,    // index dictionary for read
+FUNCTION indexdr(): Number
+    RETURN dictionary["hello"]
+END FUNCTION
+
+#    INDEXDW,    // index dictionary for write
+FUNCTION indexdw()
+    dictionary["Neon"] := 20.1797
+END FUNCTION
+
+#    INDEXDV,    // index dictionary value
+FUNCTION indexdv()
+    FUNCTION d(): Dictionary<Number>
+        RETURN dictionary
+    END FUNCTION
+    #print("\(d()[\"hello\"])")
+    print(d()["hello"].toString())
+END FUNCTION
+
+#    INA,        // in array
+FUNCTION ina()
+    IF 5 IN array THEN
+    END IF
+END FUNCTION
+
+#    IND,        // in dictionary
+FUNCTION ind()
+    IF "two" IN dictionary THEN
+    END IF
+END FUNCTION
+
+#    CALLP,      // call predefined
+FUNCTION callp()
+    print("Hello, World!")
+END FUNCTION
+
+#    CALLF,      // call function
+FUNCTION callf()
+    callp()
+END FUNCTION
+
+#    CALLMF,     // call module function
+FUNCTION callmf(): Number
+    RETURN module.function(1)
+END FUNCTION
+
+#    CALLI,      // call indirect
+FUNCTION calli()
+    FUNCTION life()
+    END FUNCTION
+    VAR pLife: FUNCTION() := life
+    pLife()
+END FUNCTION
+
+#    JUMP,       // unconditional jump
+FUNCTION jump()
+    IF 1 = 2 THEN
+    END IF
+END FUNCTION
+
+#    JF,         // jump if false
+FUNCTION jf()
+    FOR i := 1 TO 2 DO
+    END FOR
+END FUNCTION
+
+#    JT,         // jump if true
+FUNCTION jt():Boolean
+    RETURN TRUE OR FALSE
+END FUNCTION
+
+#    JFCHAIN,    // jump and drop next if false
```

Comment:
This opcode doesn't even exist any longer.

Reply:
Removed.

Reply:
Should this have generated an error?